README.md

```diff
@@ -193,4 +193,4 @@ I am grateful to all previous project reviewers for their advice. Thanks to them
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Please kindly ensure that all commented codes are removed from the following files listed below 👇 

`/app/model/purchase.rb line 4`
`/app/spec/models/`all-the-files in your model's specs 👀 and
` /app/config/routes.rb line 12`.

Reply:
Dear reviewer, thank you for pointing out. I have removed all comments as mentioned by you.